PRIVACY.md

```diff
@@ -0,0 +1,47 @@
+## Privacy Notice
+
+This privacy notice discloses the privacy practices for Flagrow Bazaar. This privacy notice applies solely to information collected by this application and its associated website at flagrow.io. It will notify of the following:
+
+* What personally identifiable information is collected from YOU through the website, how it is used and with whom it may be shared.
+* What choices are available to YOU regarding the use of your data.
+* The security procedures in place to protect the misuse of information YOU provide.
+* How YOU can correct any inaccuracies in the information.
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, YOU refers to a website that has installed Flagrow Bazaar and its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
```

Comment:
Maybe you should define *APPLICATION* first so in can be re-used in the *YOU* definition ?